.github/workflows/sql-release.yml

```diff
@@ -0,0 +1,62 @@
+name: Release SQL shared library
+on:
+  push:
+    branches: ["main"]
+    paths:
+      - ".github/workflows/sql-release.yml"
+      - "sql/**"
+jobs:
+  librillsql:
+    strategy:
+      matrix:
+        include:
+          - os: macos
+            arch: amd64
+            runner: macos-latest
+          - os: linux
+            arch: amd64
+            runner: ubuntu-latest
+          - os: windows
+            arch: amd64
+            runner: windows-latest
+    runs-on: ${{ matrix.runner }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          version: "22.1.0"
```

Comment:
can we add note sql pom.xml that these needs to be updated if version are changed in there.

Reply:
Done